pihole-google.txt

```diff
@@ -7719,3 +7719,6 @@ r3---sn-h0jeln7l.gvt.com
 r3.sn-h0jeln7l.gvt.com
 r5---sn-h0jeen7d.gvt.com
 r5.sn-h0jeen7d.gvt.com
+s.youtube.com
+#Amazon
+acsechocaptiveportal.com
```

Comment:
Amazon domains should be removed.

You can make pull request for Amazon in https://github.com/nickspaargaren/no-amazon

Reply:
Removed and I will put it in the one that tells me and if I see more from Amazon?

Reply:
Thanks for your adjustments! Ans sure, add as many as you can to https://github.com/nickspaargaren/no-amazon !